network-api/networkapi/wagtailpages/pagemodels/article_fields.py

```diff
@@ -6,15 +6,17 @@
 from wagtail.contrib.table_block.blocks import TableBlock
 
 article_fields = [
-    ('content', customblocks.ArticleRichText(
-        features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'link', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'],
-    )),
+    ('airtable', customblocks.AirTableBlock()),
     ('callout', blocks.BlockQuoteBlock(
         template="wagtailpages/blocks/article_blockquote_block.html"
     )),
-    ('table', TableBlock(
-        template="wagtailpages/blocks/article_table_block.html"
+    ('content', customblocks.ArticleRichText(
+        features=['bold', 'italic', 'h2', 'h3', 'h4', 'h5', 'link', 'large', 'ol', 'ul', 'image', 'hr', 'footnotes'],
     )),
     ('double_image', customblocks.ArticleDoubleImageBlock()),
     ('full_width_image', customblocks.ArticleFullWidthImageBlock()),
+    ('table', TableBlock(
+        template="wagtailpages/blocks/article_table_block.html"
+    )),
+    ('video', customblocks.VideoBlock()),
```

Comment:
Added two new options `airtable` and `video`. But I have also rearranged the list in alphabetically.